src/Utility/Chebyshev.h

```diff
@@ -0,0 +1,137 @@
+// $Id$
+//==============================================================================
+//!
+//! \file Chebyshev.h
+//!
+//! \date Jul 13 2020
+//!
+//! \author Arne Morten Kvarving / SINTEF
+//!
+//! \brief Evaluation of Chebyshev polynomials.
+//!
+//==============================================================================
+
+#ifndef _CHEBYSHEV_H
+#define _CHEBYSHEV_H
+
+#include "Function.h"
+#include "TensorFunction.h"
+
+#include <array>
+#include <memory>
+
+
+/*!
+  \brief Evaluation of Chebyshev polynomials.
+*/
+
+class Chebyshev
```

Comment:
Maybe this could be a namespace instead, since it only contains some static (global) functions?

Reply:
done.